games-action/nanosaur/patches/nanosaur-1.4.3.patchset

```diff
@@ -0,0 +1,84 @@
+From c5dc01605d0ded5f7e990bfe52bde69e2ef27c3b Mon Sep 17 00:00:00 2001
+From: Ivan Holmes <ivan@ivanholmes.co.uk>
+Date: Fri, 6 May 2022 22:22:59 +0100
+Subject: Add Haiku header directory so we can find SDL2
+
+
+diff --git a/cmake/FindSDL2.cmake b/cmake/FindSDL2.cmake
+index 27e773a..c9e1f09 100644
+--- a/cmake/FindSDL2.cmake
++++ b/cmake/FindSDL2.cmake
+@@ -58,7 +58,7 @@ elseif(APPLE)
+ else()
+     find_path(SDL2_INCLUDE_DIRS "SDL.h"
+         HINTS $ENV{SDL2DIR}
+-        PATH_SUFFIXES "include/SDL2" "include"
++        PATH_SUFFIXES "include/SDL2" "include" "headers/SDL2"
```

Comment:
Not checked, but I guess this would fail on 32bit, suggestion (not checked also) ...
```diff
- +        PATH_SUFFIXES "include/SDL2" "include" "headers/SDL2"
+ +        PATH_SUFFIXES "include/SDL2" "include" "headers/x86/SDL2" "headers/SDL2"
```

Reply:
I thought usually you could just add the SDL2 directory.  Like...
```
PATH_SUFFIXES "include/SDL2" "include" "SDL2"
```

Reply:
Tried it and you can indeed